tools/src/component-generator.ts

```diff
@@ -204,7 +208,8 @@ function createTemplateDto(templates: string[]) {
     ? templates.map((actualOptionName) => ({
         actualOptionName,
         render: formatTemplatePropName(actualOptionName, "Render"),
-        component: formatTemplatePropName(actualOptionName, "Component")
+        component: formatTemplatePropName(actualOptionName, "Component"),
+        keyExpr: formatTemplatePropName(actualOptionName, "ReactKeyExpr")
```

Comment:
Could we use smth shorter and simplier then `ReactKeyExpr` suffix?
`Expr` part is unnecessary and may be confusing as Expressions are used widely in the DevExtreme data binding.

What about `ReactKey`.. well, it's shorter, but personally I don't like using word React in this case. It's all React)

May be `ElementKey` or `RenderKey` or smthLike ?

Reply:
We add smth like key getter, not a key